packages/wdio-cli/src/commands/run.ts

```diff
@@ -146,23 +146,22 @@ export async function launch(wdioConfPath: string, params: Partial<RunCommandArg
 }
 
 export async function handler(argv: RunCommandArguments) {
-    const { configPath, ...params } = argv
+    const { configPath = 'wdio.conf.js', ...params } = argv
+    const configPathNoExtension = configPath.substring(0, configPath.lastIndexOf('.'))
 
-    const canAccessConfigPath = await fs.access(configPath).then(
+    const canAccessConfigPath = await fs.access(`${configPathNoExtension}.js`).then(
         () => true,
-        () => false
+        () => fs.access(`${configPathNoExtension}.ts`).then(
+            () => true, () => false
+        )
     )
+    const wdioConf = configPath.includes(process.cwd())
```

Comment:
This should be moved to the top, if `canAccessConfigPath` fails for e.g. a `wdio.conf.js` but passes for a `wdio.conf.ts` then we continue here with `wdio.conf.js`